web/source/resources/osk/kmwosk.css

```diff
@@ -409,7 +410,8 @@ div.android #keytip {background-color:#f00;}
 
 .kmw-embedded .ios.kmw-osk-frame .kmw-key-layer {position: absolute !important; top: 0 !important; bottom: initial !important;}
```

Comment:
Not sure why this isn't also excluded. I would like to remove this line as well and re-test